crates/bevy_utils/src/lib.rs

```diff
@@ -292,3 +292,13 @@ impl<F: FnOnce()> Drop for OnDrop<F> {
         callback();
     }
 }
+
+/// Like [`tracing::trace`], but conditional on cargo feature `detailed_trace`.
+#[macro_export]
+macro_rules! detailed_trace {
+    ($($tts:tt)*) => {
+        if cfg!(detailed_trace) {
```

Comment:
This could just be a `#[cfg(detailed_trace)]` block. This gives the compiler optimizer less work to do.

Reply:
I agree.

Reply:
I wrote it this way so that mistakes in using the macros (or bitrot from surrounding code changes) are caught even if the feature is not activated